static/js/notifications.js

```diff
@@ -402,6 +402,17 @@ function message_is_notifiable(message) {
         return false;
     }
 
+    // The notification should only be displayed once. To prevent multiple
+    // tabs from opening a notification each for their own, we store the
+    // notification ID in localStorage and check if a notification was
+    // already opened.
+    if (typeof(Storage) !== "undefined") {
+        if (localStorage.getItem(message.id)) {
+            return false;
+        }
+        localStorage.setItem(message.id, 'true');
+    }
```

Comment:
This looks great!  But I'm thinking maybe we want to namespace things so that it's easy to put other keys in the local storage later.  E.g. do `var key = 'notification:' + message_id` and then use that as the index in localStorage.  What do you think?